src/pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -31,7 +31,7 @@
     </Dependency>
 
     <ProjectReference Include="..\Microsoft.Net.UWPCoreRuntimeSdk\Microsoft.Net.UWPCoreRuntimeSdk.pkgproj">
-      <TargetFramework>uap10.1</TargetFramework>
+      <TargetFramework>uap10.0.15138</TargetFramework>
```

Comment:
VS is using the moniker "10.0.xxxxx" where xxxxx is the Windows build number. To complete these changes down the road, we will set this value to the exact value of the RS3 build number.